e2e/guardrails/basal_rate_schedules.js

```diff
@@ -0,0 +1,61 @@
+const { limits } = require('../../src/index');
+
+var basalRateSchedules = (test) => {
+    it('can set max limit', async () => {
+        await test.settingsScreen.SetBasalRates(
+            [{ time: '12:00 AM', unitsPerHour: limits.basalRates.max.limit }],
+            { fromSettings: true, toSettings: false }
+        );
+        //TODO assert on warning
+    });
+    it('can set max warning', async () => {
+        await test.settingsScreen.SetBasalRates(
+            [{ time: '12:30 AM', unitsPerHour: limits.basalRates.max.warning - limits.basalRates.step }],
```

Comment:
[nit] Now I am unclear on what `SetBasalRates` actually does? Does it *set* the basal rates to the specified schedule, or does it *add* to the already existing basal rate schedule?

Also, might want to consider `SetBasalRateSchedule`.